src/gc/sample/gcenv.h

```diff
@@ -105,7 +105,7 @@ class Thread
 
     bool PreemptiveGCDisabled()
     {
-        return !!m_fPreemptiveGCDisabled;
+        return m_fPreemptiveGCDisabled != 0;
```

Comment:
It would look better to change type of `m_fPreemptiveGCDisabled` to bool in this file.